### ISSUE TITLE
Unable to see predictions on Labeling tab after training model and uploading new data

### ISSUE THREAD

**Issue by hitz02** [heartexlabs/label-studio#483]
Dear Team,

Thanks for this amazing tool!

While trying my hands on the NER annotation, I encountered an issue.
Or maybe I am not sure if I am using it correctly.

Issue - Predictions are not visible on the labeling tab (in the labeling region)

Although, I can see on the right-hand side a model entry in the predictions section but not on the task on which the prediction is performed.
I can also see the predictions key-value pair in -> tasks tab -> show task data (by clicking on </> icon)

Steps performed - 
1. Started NER ML backend
2. Started Label studio
3. Choose NER task from setup
4. Modified View as per my requirement
4. Uploaded CSV file containing annotation data
5. Completed a few labeling
6. From the model tab, clicked start training
7. It showed Training in Progress for some time (model download + 100 epochs)
(Possible bug - We need to manually refresh the page once the training is done, it doesn't auto-reload on this step)
8. uploaded new training data
9. The tasks tab showed an incremented count of tasks
10. On opening one of the tasks in the labeling tab, **failed** to see the text highlighted

Let me know if this is already reported and refer me to the issue id
Or let me know if I am missing any step.

Thanks!

**Comment by makseq** [comment]
You need to add some "text": "..." to the each item of prediction.result. 

```
{
  "data": {
    "text": "W.K KELLOGG CRUNCHY MUESLI COCONUT&CASHEW BOX 01 ER 00400 G"
  },
  "id": 5,
  "predictions": [
    {
      "cluster": null,
      "created_by": "my-ner-project9480",
      "created_date": "2020-11-06T12:39:15.777223",
      "result": [
        {
          "from_name": "label",
          "id": "xxxx1",
          "to_name": "text",
          "type": "labels",
          "value": {
            "end": 3,
            "labels": [
              "ZUSATZMARKE"
            ],
            "start": 0,
            ---> "text": "..."  <----
          }
        }
      ],
      "score": 5.3785624504089355
    }
  ]
}
```

**Comment by hitz02** [comment]
The issue was resolved after @makseq added a fix [here](https://github.com/heartexlabs/label-studio-transformers/blob/master/models/ner.py) for adding **text:value** pair as described above.
